SPDY/SPDYCacheStoragePolicy.m

```diff
@@ -13,6 +13,59 @@
 
 #import "SPDYCacheStoragePolicy.h"
 
+typedef struct _HTTPTimeFormatInfo {
+    const char *readFormat;
+    const char *writeFormat;
+    BOOL usesHasTimezoneInfo;
+} HTTPTimeFormatInfo;
+
+static HTTPTimeFormatInfo kTimeFormatInfos[] =
+{
+    { "%a, %d %b %Y %H:%M:%S %Z", "%a, %d %b %Y %H:%M:%S GMT", YES }, // Sun, 06 Nov 1994 08:49:37 GMT  ; RFC 822, updated by RFC 1123
+    { "%A, %d-%b-%y %H:%M:%S %Z", "%A, %d-%b-%y %H:%M:%S GMT", YES }, // Sunday, 06-Nov-94 08:49:37 GMT ; RFC 850, obsoleted by RFC 1036
+    { "%a %b %e %H:%M:%S %Y", "%a %b %e %H:%M:%S %Y", NO },           // Sun Nov  6 08:49:37 1994       ; ANSI C's asctime() format
+};
+
+
+static NSDate *HTTPDateFromString(NSString *string)
+{
+    NSDate *date = nil;
+    if (string) {
+        struct tm parsedTime;
+        const char *utf8String = [string UTF8String];
+
+        for (int format = 0; (size_t)format < (sizeof(kTimeFormatInfos) / sizeof(kTimeFormatInfos[0])); format++) {
+            HTTPTimeFormatInfo info = kTimeFormatInfos[format];
+            if (info.readFormat != NULL && strptime(utf8String, info.readFormat, &parsedTime)) {
```

Comment:
`strptime(3)` doesn't initialize `struct tm`, should be clear before using it, or, some of value may be undefined.
`strptime(3)` using current system locale, which is unknown. must be `strptime_l(3)` ?